optuna/trial/_trial.py

```diff
@@ -597,6 +597,7 @@ def _suggest(self, name: str, distribution: BaseDistribution) -> Any:
                     study, trial, name, distribution
                 )
 
+            # param_value is validated here (invalid value like np.nan raises ValueError)
```

Comment:
```suggestion
            # `param_value` is validated here (invalid value like np.nan raises ValueError).
```
Please also refer to https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#optuna-specific-conventions.